cloudify_aws/ecs/service.py

```diff
@@ -0,0 +1,320 @@
+########
+# Copyright (c) 2015 GigaSpaces Technologies Ltd. All rights reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+#    * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    * See the License for the specific language governing permissions and
+#    * limitations under the License.
+
+# Stdlib imports
+
+# Third party imports
+
+# Cloudify imports
+from cloudify import ctx
+from cloudify.decorators import operation
+from cloudify.exceptions import NonRecoverableError
+
+# This package imports
+from cloudify_aws import constants
+from cloudify_aws.base import AwsBaseNode
+from cloudify_aws.connection import EC2ConnectionClient
+
+
+@operation
+def create(args=None, **_):
+    return Service().created(args)
+
+
+@operation
+def delete(args=None, **_):
+    return Service().deleted(args)
+
+
+class Service(AwsBaseNode):
+    def __init__(self, client=None):
+        client = client or EC2ConnectionClient().client3('ecs')
+
+        super(Service, self).__init__(
+            constants.ECS_SERVICE['AWS_RESOURCE_TYPE'],
+            constants.ECS_SERVICE['REQUIRED_PROPERTIES'],
+            client=client,
+            resource_id_key=constants.ECS_SERVICE['RESOURCE_ID_KEY'],
+        )
+
+        self.not_found_error = constants.ECS_SERVICE['NOT_FOUND_ERROR']
+
+    def get_relationship_target(self, relationships, intended_relationship):
```

Comment:
I'll take a look at these this evening and migrate them to wherever they seem to fit best.